test/e2e/scrapeconfig_test.go

```diff
@@ -241,8 +241,6 @@ func testPromOperatorStartsWithoutScrapeConfigCRD(t *testing.T) {
 
 // testScrapeConfigKubernetesNodeRole tests whether Kubernetes node monitoring works as expected
 func testScrapeConfigKubernetesNodeRole(t *testing.T) {
-	skipPrometheusTests(t)
```

Comment:
This was culprit for skipping e2e test which didn't catch regression due to https://github.com/prometheus-operator/prometheus-operator/pull/5709/files#diff-453ce9c721c51bf9591c2b212e9ce50157b9c40d1b211177bf502be268099656L2298